libraries/standard/mqtt/src/mqtt.c

```diff
@@ -214,6 +245,107 @@ static MQTTStatus_t sendPublish( const MQTTContext_t * const pContext,
 
 /*-----------------------------------------------------------*/
 
+static uint32_t calculateElapsedTime( uint32_t later,
+                                      uint32_t start )
+{
+    return later - start;
+}
+
+/*-----------------------------------------------------------*/
+
+static MQTTStatus_t receivePacket( MQTTContext_t * const pContext,
+                                   MQTTPacketInfo_t incomingPacket,
+                                   uint32_t remainingTimeMs )
+{
+    return MQTTSuccess;
+}
+
+/*-----------------------------------------------------------*/
+
+static MQTTStatus_t receiveConnack( MQTTContext_t * const pContext,
+                                    uint32_t timeoutMs,
+                                    MQTTPacketInfo_t * const incomingPacket,
+                                    bool * const pSessionPresent )
+{
+    MQTTStatus_t status = MQTTSuccess;
+    MQTTGetCurrentTimeFunc_t getTimeStamp = NULL;
+    uint32_t entryTime = 0U, remainingTimeMs = 0U, timeTaken = 0U;
```

Comment:
All time values should have units in the name, e.g. `entryTimeMs`, `timeTakenMs`.

Reply:
Added it